locales/it/profile/profile_account_main.md

```diff
@@ -1,2 +1,3 @@
 **Nella sezione Profilo puoi gestire tutte le tue impostazioni.**
```

Comment:
> **Nella sezione Profilo puoi gestire tutte le tue impostazioni.**

Do I read "Profilo" here because we are waiting for the copy review?

Reply:
As you can see from the last line of this file (//edit) that you commented on a couple of comments above, this file is awaiting review copy

Reply:
There is too the label "copy review needed" 😁

Reply:
I saw the label, just checking in.